### DISCUSSION TITLE
Unable to send mail from mailclient

### DISCUSSION THREAD

**Discussion by jhjacobs81** [modoboa/modoboa#2639] [General]
I remember i had this problem in the past, but i'm not sure what and how..

I setup a standard MODOBOA server on Debian 10, i can logon to the webmail and send email to everyone. But when i try to use Thunderbird i get an error: 550 5.1.1 recipient address rejected user unknown

does anyone have solved this?

**Comment by tonioo** [comment]
@jhjacobs81 I guess you need to refresh or delete your verify cache

**Comment by jhjacobs81** [comment]
how and why would i do that? 

this is the full error lines:
```
Oct 13 12:27:53 localhost postfix/submission/smtpd[2016]: warning: problem talking to server 127.0.0.1:9999: Success
Oct 13 12:27:53 localhost postfix/submission/smtpd[2016]: NOQUEUE: reject: RCPT from dhcp-077-249-089-140.chello.nl[77.249.89.140]: 451 4.3.5 <sendto@thisaddress.com>: Recipient address rejected: Server configuration problem; from=<sendfrom@thisaddress.com> to=<sendto@thisaddress.com> proto=ESMTP helo=<[192.168.178.116]>
```
